contracts/VRF.sol

```diff
@@ -29,7 +29,8 @@ library VRF {
   // Order of the curve
   uint256 public constant NN = 0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364141;
 
-  /// @dev Public key derivation from private key.
+  /// @notice Public key derivation from private key.
+  /// Warning: this function should not be used to derive your public key as it would expose the private key
```

Comment:
```suggestion
  /// @dev Public key derivation from private key.
  /// Warning: this function should not be used to derive your public key as it would expose the private key.
```